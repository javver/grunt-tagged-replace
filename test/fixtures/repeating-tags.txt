repeating tag 1 /*tag*/ /*/tag*/ end repeat tag 1
repeating tag 2 /*tag*/ to be replaced /*/tag*/ end repeat tag 2
