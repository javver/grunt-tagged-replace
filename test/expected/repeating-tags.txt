repeating tag 1 /*tag*/ replaced content /*/tag*/ end repeat tag 1
repeating tag 2 /*tag*/ replaced content /*/tag*/ end repeat tag 2
